context/app/static/js/components/search/Facets/FilterChips.tsx

```diff
@@ -52,6 +59,268 @@ function FilterChip({ onDelete, label, ...props }: ChipProps & { onDelete: () =>
   );
 }
 
+interface MultiValueFilterChipProps {
+  field: string;
+  values: string[];
+  onDeleteValue: (value: string) => void;
+}
+
+function MultiValueFilterChip({ field, values, onDeleteValue }: MultiValueFilterChipProps) {
+  const analyticsCategory = useSearchStore((state) => state.analyticsCategory);
+  const getFieldLabel = useGetFieldLabel();
+  const getTransformedFieldValue = useGetTransformedFieldValue();
+  const anchorEl = useRef<HTMLDivElement>(null);
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  const fieldLabel = getFieldLabel(field);
+  const chipLabel =
+    values.length === 1 ? `${fieldLabel}: ${getTransformedFieldValue({ field, value: values[0] })}` : fieldLabel;
+
+  const handleChipClick = useCallback(() => {
+    if (values.length > 1) {
+      setMenuOpen(true);
+    }
+  }, [values.length]);
+
+  const handleMenuClose = useCallback(() => {
+    setMenuOpen(false);
+  }, []);
+
+  const handleDeleteValue = useCallback(
+    (value: string) => {
+      onDeleteValue(value);
+      trackEvent({
+        category: analyticsCategory,
+        action: 'Unselect Facet Chip',
+        label: `${fieldLabel}: ${getTransformedFieldValue({ field, value })}`,
+      });
+      // Close menu if this was the last value
+      if (values.length === 1) {
+        setMenuOpen(false);
+      }
+    },
+    [onDeleteValue, analyticsCategory, fieldLabel, field, getTransformedFieldValue, values.length],
+  );
+
+  const handleDeleteAll = useCallback(() => {
+    values.forEach((value) => onDeleteValue(value));
+    trackEvent({
+      category: analyticsCategory,
+      action: 'Unselect Facet Chip',
+      label: chipLabel,
+    });
+    setMenuOpen(false);
+  }, [onDeleteValue, values, analyticsCategory, chipLabel]);
+
+  // For single values, behave like a regular chip
+  if (values.length === 1) {
+    return <FilterChip label={chipLabel} onDelete={() => handleDeleteValue(values[0])} />;
+  }
```

Comment:
Could we create separate components for multi and single values so we don't have to always instantiate all of the hooks for the multi value cases?